cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -304,6 +309,16 @@ func (s *CMServer) Run(_ []string) error {
 	}
 	pvRecycler.Run()
 
+	if s.EnablePersistentVolumeProvisioner {
+		if createableVolumePlugin := getCreatableVolumePluginForCloud(cloud, s); createableVolumePlugin != nil {
```

Comment:
So, there's going to be only one type of PV any cluster can provision, @markturansky ?